### DISCUSSION TITLE
Point DNSCrypt to Local on Premise Server

### DISCUSSION THREAD

**Discussion by connortess** [DNSCrypt/dnscrypt-proxy#1934] [Q&A]
Is there a way to point DNSCrypt to a plain text DNS resolver that is on a local network? The goal is to send all traffic that DNSCrypt would see to alocal dns resolver instead of all the included servers that are provided in the dnscrypt-proxy.toml that use DNSCrypt, DoH, and ODoH.

**Comment by jedisct1** [comment]
https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Forwarding
